packages/api-explorer/src/JsonForm/custom-bootstrap4.css

```diff
@@ -8,3 +8,36 @@
     margin: 5px;
 }
 
+.card-title label {
+    font-size: 18px;
+    color: #666666;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+.mia-container-wrapper {
+    border-radius: 4px;
+    border: solid 1px #d8d8d8;
+    background: #fafafa;
+    padding: 5px
+}
+
+.form-group {
```

Comment:
let's better separate our styles from the ones of jsonEditor and antd (maybe use comments?)

Reply:
done